### ISSUE TITLE
Local videos not getting copied as images

### ISSUE THREAD

**Issue by krishvk** [sphinx-contrib/video#33]
I have a nested hierarchy and local videos, that do not work as they are not getting copied as images are. Let me know if you need small env to reproduce.

**Comment by Calinou** [comment]
I believe this is a Sphinx limitation – images are only included in a top-level `_images` folder as they're referenced by a `.. image::` directive. I'm not sure how it could be worked around from the extension side.

**Comment by krishvk** [comment]
When we include figure/images, we refer it locally, when build I see the image is getting copied to _images folder and linked accordingly. This is just not happening for videos.

 

As I have pointed out, image reference is working but video reference is not

**Comment by jdillard** [comment]
You can use reference from the top level, such as:
```
.. video:: /_static/movie.mp4
```

**Comment by krishvk** [comment]
True, but that feels more like a workaround and not a fix. Since image work but videos do not.
